### PR TITLE
Add a share button when WCs are found

### PR DESCRIPTION
**Pull request by jlengrand**

### CODE REVIEW

**Comment by jlengrand** [comment]
Fixes #7

**Comment by jlengrand** [comment]
This will contain the changes of #10 in it, since that's what I'm using. Changing the target branch for that reason for now.

@thepassle AFAIK this is a first working version, though it will need #12 to pick the domain up. I placed the Twitter thing at the end so it's not too much in your face (not ideal though, for large website like youtube it's not likely people will see it).

Waiting for feedback before I keep working on it :).

**Comment by jlengrand** [comment]
NEvermind, seems like I can't point the PR to #10 . You'll have to ignore those changes when checking out the code

File: src/share-element.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    constructor() {
+        super();
+        this.locatorUrl = encodeURI("https://chrome.google.com/webstore/detail/custom-elements-locator/eccplgjbdhhakefbjfibfhocbmjpkafc");
+    }
+
+    render() {
+        const tweetText = encodeURI(`I just found some custom-elements on ${this.domain} using locator!`);
+        return html`
+            <div class="share">
+                <img class="icon" src="./share.svg"/>
```
Comment (thepassle) [inline src/share-element.js]:
```suggestion
                <img alt="" class="icon" src="./share.svg"/>
```

or `<title>` in the svg 🙂

**Comment by thepassle** [comment]
should we merge this in https://github.com/open-wc/locator/pull/12 instead of master?

File: src/main.js [hunk 2]
```diff
@@ -50,6 +51,7 @@ class CustomElementsLocator extends LitElement {
       ${this.loaded
         ? this.customElements.length > 0
           ? html`
+              <h2>I found custom elements on </h2>
```
Comment (thepassle) [inline src/main.js]: I think we can remove this line

**Comment by thepassle** [comment]
Should we just close https://github.com/open-wc/locator/pull/10 in favor of this?

File: src/share-element.js (hunk 1 above)
Comment (jlengrand) [reply src/share-element.js]: good point

File: src/main.js (hunk 2 above)
Comment (jlengrand) [reply src/main.js]: I considered it this afternoon. But without that line, will it be obvious enough?

**Comment by jlengrand** [comment]
I'm all up for merging whatever into whatever to simplify the merging process :). Was trying to separate to help with the review

**Comment by thepassle** [comment]
Ok, lets close #10, because those changes are in here, and lets merge this into #12 because it depends on the domain name from there

**Comment by jlengrand** [comment]
#10 is now closed.

**Comment by jlengrand** [comment]
And comments here processed.

**Comment by jlengrand** [comment]
I don't think I can merge into #12 myself though can i?

**Comment by jlengrand** [comment]
Ok, give me a few I'll clean that PR up and point it towards #12

File: src/main.js (hunk 2 above)
Comment (jlengrand) [reply src/main.js]: Done

**Comment by jlengrand** [comment]
Alright, ready to merge on my end

**Review by thepassle — APPROVED** [review]

**Comment by thepassle** [comment]
Very nice changes 🙂 thanks for the contribution ❤️

**Pull request merged**